README.md

```diff
@@ -281,9 +315,15 @@ Next, let's find the total uniques for a given ad, grouped by geography:
 select sum(uniques) AS totalUniques, geo from aggrAdImpressions where publisher='publisher11' group by geo order by totalUniques desc limit 20;
 ```
 
-Now that we've seen some standard OLAP queries over the exact data, let's execute the same queries on our sample tables using SnappyData's [Approximate Query Processing techinques](https://github.com/SnappyDataInc/snappydata/blob/master/docs/aqp.md). In most production situations, the latency difference here would be significant because the volume of data in the exact table would be much higher than the sample tables. Since this is an example, there will not be a significant difference; we are showcasing how easy AQP is to use.
+Now that we've seen some standard OLAP queries over the exact data, let's execute the same queries on our sample tables
+using SnappyData's [Approximate Query Processing techinques](https://github.com/SnappyDataInc/snappydata/blob/master/docs/aqp.md).
+In most production situations, the latency difference here would be significant because the volume of data in the exact
+table would be much higher than the sample tables. Since this is an example, there will not be a significant difference;
+we are showcasing how easy AQP is to use.
 
-We are asking for an error rate of 20% or below and a confidence interval of 0.95 (note the last two clauses on the query). The addition of these last two clauses route the query to the sample table despite the exact table being in the FROM clause. If the error rate exceeds 20% an exception will be produced:
+We are asking for an error rate of 20% or below and a confidence interval of 0.95 (note the last two clauses on the query).
+The addition of these last two clauses route the query to the sample table despite the exact table being in the FROM
+clause. If the error rate exceeds 20% an exception will be produced:
 
 ```sql
 select count(*) AS adCount, geo from aggrAdImpressions group by geo order by adCount desc limit 20 with error 0.20 confidence 0.95 ;
```

Comment:
Should we mention the significance of the keywords "WITH ERROR" and "CONFIDENCE"?

Reply:
Already explained in the paragraph above the query.